RELEASE.md

```diff
@@ -207,15 +207,24 @@ End-to-end tests run daily in [github.com/slsa-framework/example-package/.github
 
 ## Update verifier
 
-The next step is to update the verifier's e2e tests. For this, you need to:
+The next step is to update the verifier's GitHub Actions e2e tests. There are GitHub actions Go and generic actions.
 
-1. Generate binaries and provenance for a project, using the [example-package](https://github.com/slsa-framework/example-package) builder. You will need to create provenance via a `workflow_dispatch` from this [workflow](https://github.com/slsa-framework/example-package/blob/main/.github/workflows/e2e.go.workflow_dispatch.main.config-noldflags.slsa3.yml) of the e2e test repository, and push a tag of the form vX and vX.Y to trigger this [workflow](https://github.com/slsa-framework/example-package/blob/main/.github/workflows/e2e.go.tag.main.config-ldflags-assets.major.slsa3.yml).
+<!-- TODO: Describe GHA generic container e2e tests. -->
```

Comment:
nit: create an issue and add to the generic container milestone

Reply:
done! linked here: made two issues, one for creation in example-package, and one for docs linked here.